Backend/index.js

```diff
@@ -61,6 +101,39 @@ const verifyToken = (req, res, next) => {
   }
 };
 
+const updateImportantNotificationTypes = async () => {
+  const threshold = 5;
```

Comment:
i'd have this as a constant at the top of the file so that if you have to change it you only have to change it in one place rather than every function you define it in